packages/cli/src/module/generate-spec.ts

```diff
@@ -7,6 +7,13 @@ import { SpecGenerator2 } from '../swagger/specGenerator2';
 import { SpecGenerator3 } from '../swagger/specGenerator3';
 import { fsExists, fsMkDir, fsWriteFile } from '../utils/fs';
 
+export const getSwaggerOutputPath = (swaggerConfig: ExtendedSpecConfig) => {
+  const ext = swaggerConfig.yaml ? 'yaml' : 'json';
+  const specFileBaseName = swaggerConfig.specFileBaseName || 'openapi';
```

Comment:
```suggestion
  const specFileBaseName = swaggerConfig.specFileBaseName || 'swagger';
```

For Context: The previous PR was opened right before a major release (for type aliases), so I felt comfortable to make a backwards-incompatible change. If we have no other reason to make a major bump, I'd keep the current default name and merge in time for the next minor.